src/app/common/hooks/use-bitcoin-contracts.ts

```diff
@@ -196,46 +191,8 @@ export function useBitcoinContracts() {
     close();
   }
 
-  async function getHealthyDlcLinkAttestor(): Promise<string> {
```

Comment:
Since attestors are not required for the Bitcoin Contract interface, this function became unnecessary.